src/lib.rs

```diff
@@ -18,6 +18,8 @@
 //!
 //! Now when you load files with the `.libgdx.atlas` extension through the asset server, or even `bevy_asset_loader`, they will load as a [`LibGdxAtlasAsset`] which you can then use.
 
+#![deny(clippy::unwrap_used, clippy::panic, clippy::expect_used)]
```

Comment:
lets use workspace lints instead

Reply:
This is a nightly feature, do you wish for us to switch to nightly for this crate? https://doc.rust-lang.org/cargo/reference/unstable.html#lintscargo

Reply:
huh? thats stable:
```
[workspace.lints.clippy]
all = "deny"
unwrap_used = "deny"
```

Reply:
Ah, okay. So turns out the tests (which I was removing the allow attribute from) weren't triggering this at all, even tho I remember doing that. Pushing this now. (Sorry for the delay due to my incomprehension of things, again).

Reply:
no worries!